nav2_loopback_sim/nav2_loopback_sim/loopback_simulator.py

```diff
@@ -209,6 +230,143 @@ def debug(self, msg):
         self.get_logger().debug(msg)
         return
 
+    def get_map(self):
+        request = GetMap.Request()
+        if self.map_client.wait_for_service(timeout_sec=1.0):
```

Comment:
```suggestion
        if self.map_client.wait_for_service(timeout_sec=5.0):
```

if we're only trying once, we should try hard